dataland-frontend/tests/e2e/specs/eu-taxonomy-financials/DataIntegrity.ts

```diff
@@ -35,9 +31,15 @@ describeIf(
     });
 
     function getPreparedFixture(name: string): FixtureData<EuTaxonomyDataForFinancials> {
-      return preparedFixtures.find((it): boolean => it.companyInformation.companyName == name)!;
+      const preparedFixture = preparedFixtures.find((it): boolean => it.companyInformation.companyName == name)!;
+      if (!preparedFixture) {
+        throw new Error("The provided company name could not be found in the prepared fixtures.");
```

Comment:
Agree.  I will check now for a better Error Code.

Reply:
In the typescript docs they throw a "ReferenceError" if a variable is not defined.
Ok, to befair: In our case here, "preparedFixture" is always defined, but it can bear the value "undefined".
So I changed the error to a ReferenceError because it might make more sense than just an Error.
Is this ok?